google/cloud/spanner/value.cc

```diff
@@ -88,6 +90,52 @@ bool Equal(google::spanner::v1::Type const& pt1,
   }
 }
 
+// A 256-element table mapping all `unsigned char` byte values to a printable
+// string. All naturally printable values are used directly. All non-printable
+// bytes use their octal escape sequence. Whitespace characters use their short
+// form like '\t' and '\n'.
```

Comment:
I don't think "whitespace characters" is an accurate characterization here.  It doesn't include SPACE for example, and I wouldn't say that `\a` counts as whitespace.

Reply:
Fixed.